client/state/stats/lists/utils.js

```diff
@@ -101,12 +101,12 @@ function parseAvatar( avatarUrl ) {
  * @returns {Array}         CSV Row
  */
 export function buildExportArray( data, parent = null ) {
-	if ( ! data || ! data.label || ! data.value ) {
+	if ( ! data || ! data.label || ! data.value || ! data.actions ) {
```

Comment:
Should we check `data.actions.length` as well, or is it safe to assume that if it exists it will always include at least one element?

Reply:
I believe that there'll always be one element, but I added a commit with that check just in case! :)